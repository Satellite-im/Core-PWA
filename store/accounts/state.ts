import { AccountsState } from './types'

const InitalAccountsState: AccountsState = {
  locked: true,
  error: '',
  pinHash: '',
  active: '',
  gasPrice: '',
  phrase: '',
  encryptedPhrase: '',
}

export default InitalAccountsState
