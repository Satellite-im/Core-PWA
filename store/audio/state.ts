interface AudioState {
  muted: Boolean
  deafened: Boolean
  volume: Number
  inputVolume: Number
  previousVolume: Number
  sounds: {
    inboundMedia: Number
    outboundMedia: Number
    system: Number
  }
}

const InitalAudioState = (): AudioState => ({
  muted: false,
  deafened: false,
  volume: 100,
  inputVolume: 100,
  previousVolume: 0,
  sounds: {
    inboundMedia: 100,
    outboundMedia: 100,
    system: 100,
  },
})

export default InitalAudioState
