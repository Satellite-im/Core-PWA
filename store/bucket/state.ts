import { BucketState } from './types'

const InitialBucketState = (): BucketState => ({
  initialized: false,
  fileSystem: null,
})

export default InitialBucketState
