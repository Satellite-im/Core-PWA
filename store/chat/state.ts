import { ChatState } from './types'
import { ScrollDirections } from '~/types/chat/chat'

export const initialCurrentChat = {
  messages: [],
  page: 1,
  size: 10,
  hasNextPage: true,
  direction: ScrollDirections.TOP,
  isMessagesLoading: false,
  lastLoadedMessageId: '',
  isScrollOver: true,
  offset: 0,
  showOlderMessagesInfo: false,
}

const InitialChatState = (): ChatState => ({
  replies: [],
  chatTexts: [],
  files: {},
  countError: false,
  currentChat: initialCurrentChat,
  uploadProgress: {},
})

export default InitialChatState
