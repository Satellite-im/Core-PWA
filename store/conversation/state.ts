import { ConversationState } from './types'

export default function InitialConversationState(): ConversationState {
  return {
    id: '',
    type: 'friend',
    calling: false,
    participants: [],
  }
}
