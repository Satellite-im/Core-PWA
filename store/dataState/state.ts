export enum DataStateType {
  Empty = 'E',
  Loading = 'L',
  Updating = 'U',
  Ready = 'R',
}
interface DataState {
  files: DataStateType
  friends: DataStateType
  search: DataStateType
}

const InitialDataState: DataState = {
  files: DataStateType.Empty,
  friends: DataStateType.Empty,
  search: DataStateType.Empty,
}

export default InitialDataState
