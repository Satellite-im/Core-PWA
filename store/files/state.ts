import { FilesState } from './types'

const InitalFilesState = (): FilesState => ({
  tree: [],
})

export default InitalFilesState
