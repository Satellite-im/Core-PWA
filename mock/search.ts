import { FileType, Folder } from '~/types/files/file'

export const Search = {
  from: [
    {
      name: 'Taurus Nix',
      address: '0xdf9eb223bafbe5c5271415c75aecd68c21fe3d7f',
      status: 'Some important status message',
      state: 'online',
    },
    {
      name: 'Halley Themis',
      address: '0x07ee55aa48bb72dcc6e9d78256648910de513eca',
      status: 'Working on the space station',
      state: 'offline',
      last_message:
        'At the edge of forever bits of moving fluff radio telescope the carbon in our apple pies are creatures of the cosmos rings of Uranus.',
    },
  ],
  mentions: [
    {
      user: {
        name: 'Taurus Nix',
        address: '0xdf9eb223bafbe5c5271415c75aecd68c21fe3d7f',
        status: 'Some important status message',
        state: 'online',
      },
    },
  ],
}
