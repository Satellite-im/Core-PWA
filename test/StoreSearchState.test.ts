import * as state from '~/store/search/state'

describe("state.default", () => {
    test("0", () => {
        let result: any = state.default()
        expect(result).toMatchSnapshot()
    })
})
