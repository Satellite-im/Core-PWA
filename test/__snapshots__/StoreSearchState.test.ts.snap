// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`state.default 0 1`] = `
Object {
  "query": "",
  "result": null,
}
`;
